pom.xml

```diff
@@ -124,8 +130,7 @@
             <exclude>org/folio/rest/impl/*Test.java</exclude>
           </excludes>
           <includes>
-            <include>org/folio/rest/impl/StorageTestSuite.java</include>
-            <include>org/folio/rest/**</include>
+            <include>org/folio/rest/impl/ApiTestSuite.java</include>
```

Comment:
Might be better to have `ApiTestSuite.class` because 

> Regex matches are done over *.class files and not *.java files
https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html

Reply:
According to the documentation this is applicable only for regular expressions i.e. when `%regex[expr]` is used

Reply:
Okay. Also, I was looking into another project such as mod-orders and we are using `.class` https://github.com/folio-org/mod-orders/blob/master/pom.xml#L592 Not sure how much will it impact build process timing but we could try to be consistent.

Reply:
changed to .class